base_geoengine/fields.py

```diff
@@ -170,6 +191,30 @@ class GeoLine(GeoField):
     type = 'geo_line'
     geo_type = 'LINESTRING'
 
+    @classmethod
+    def from_points(cls, cr, point1, point2):
+        """
+        Converts given points in parameter to a line.
+        :param cr: DB cursor
+        :param point1: Point (BaseGeometry)
+        :param point2: Point (BaseGeometry)
+        :return: LINESTRING Object
+        """
+        sql = """
+        SELECT
+            ST_MakeLine(
+                ST_GeomFromText(%(wkt1)s, %(srid)s),
+                ST_GeomFromText(%(wkt2)s, %(srid)s)
+            )
+        """
+        cr.execute(sql, {
+            'wkt1': point1.wkt,
+            'wkt2': point2.wkt,
+            'srid': cls._slots['srid'],
```

Comment:
@benwillig `srid` can differ by instance. IMO it would be better to add `srid=None` as parameter and if not specified, takes the default one on the class.